frontend/src/metabase/dashboard/containers/DashboardApp.jsx

```diff
@@ -121,23 +124,23 @@ const DashboardApp = props => {
     if (!isRunning) {
       setIsShowingToaster(false);
     }
-    if (!isRunning && shouldSendNotification) {
+    if (
+      !isRunning &&
+      isLoadingComplete &&
+      Notification.permission === "granted"
+    ) {
```

Comment:
Ain't `isRunning` and `isLoadingComplete` connected here? I wonder if we can check for just one of them?

Reply:
Simplified with new commit